src/runtimes/node/bundlers/types.ts

```diff
@@ -0,0 +1,70 @@
+import type { Message } from '@netlify/esbuild'
```

Comment:
I had to extract all the types from `index.js` into `types.js`, to avoid a cycle dependency.